<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Alternative Enzymes Selection - Alebricks</title>
    <link href="../../css/bootstrap.css" rel="stylesheet">
    <link href="../../css/general.css" rel="stylesheet">
</head>
<body>
    <div style="background-color: #F8F9FA;">
        <div class="container"> <!-- NavBar-->
            <nav class="navbar navbar-expand-lg fixed-top bg-body-tertiary">
                <div class="container-fluid">
                    <a class="navbar-brand" href="../../index.html">Alebricks</a>
                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
                        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                        <span class="navbar-toggler-icon"></span>
                    </button>
                    <div class="collapse navbar-collapse" id="navbarSupportedContent">
                        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
                            <li class="nav-item">
                                <!-- <a class="nav-link active" aria-current="page" href="#">Home</a> -->
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">About</a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../about/members.html"> Members</a></li>
                                    <li><a class="dropdown-item" href="../about/mission_vision.html">Mission & Vision</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Project
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../project/description.html">Description</a></li>
                                    <li><a class="dropdown-item" href="../project/roadmap.html">Roadmap</a></li>
                                    <li><a class="dropdown-item" href="../project/engineering_success.html">Engineering success</a></li>
                                    <li><a class="dropdown-item" href="../project/business_model.html">Business model</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Safety
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../safety/biosafety.html">Biosafety & Biosecurity</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle disabled" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Wet & Field Lab
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item disabled" href="#">Parts</a></li>
                                    <li><a class="dropdown-item disabled" href="#">Design</a></li>
                                    <li><a class="dropdown-item disabled" href="#">Citotoxicity</a></li>
                                    <li><a class="dropdown-item disabled" href="#">Phytotoxicity</a></li>
                                    <li><a class="dropdown-item disabled" href="#">Field Tests</a></li>
                                    <li><a class="dropdown-item disabled" href="#">Future perspectives and troubleshooting</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Dry Lab
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="./Overview.html">Overview</a></li>
                                    <li><a class="dropdown-item" href="./genome_annotation.html">Genome annotation</a></li>
                                    <li><a class="dropdown-item" href="./differential-expression-analysis.html">Differential expression analysis</a></li>
                                    <li><a class="dropdown-item" href="./docking.html">Docking</a></li>
                                    <li><a class="dropdown-item" href="./alternative-enzymes-selection.html">Alternative enzymes selection</a></li>
                                    <li><a class="dropdown-item" href="./docking-mds.html">Docking and MDS</a></li>
                                    <li><a class="dropdown-item" href="./model.html">Math-Model</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle disabled" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Human Practices
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="#">Human Practices</a></li>
                                    <li><a class="dropdown-item" href="#">Integrated Human Practices</a></li>
                                    <li><a class="dropdown-item" href="#">Diversity and Inclusivity</a></li>
                                    <li><a class="dropdown-item" href="#">Education</a></li>
                                    <li><a class="dropdown-item" href="#">Philosophy and episthemology</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle disabled" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    ARTS & DESIGN
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="#">Gallery</a></li>
                                    <li><a class="dropdown-item" href="#">Videogame</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle disabled" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    AKNOWLEDGEMENTS
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="#">Attributions</a></li>
                                    <li><a class="dropdown-item" href="#">Sponsors</a></li>
                                    <li><a class="dropdown-item" href="#">Crowdfunding</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle disabled" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Sustainable development
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="#">Ecological Niche Modeling</a></li>
                                    <li><a class="dropdown-item" href="#">SDG's</a></li>
                                    <li><a class="dropdown-item" href="#">Negative Impact</a></li>
                                    <li><a class="dropdown-item" href="#">Why to protect maize?</a></li>
                                    <li><a class="dropdown-item" href="#">Climaton</a></li>
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>
            </nav>
        </div>
    </div> <!-- End Container NavBar-->

    <div class="container my-5"> <!-- Body -->
        <h1 class="text-center my-5">Alternative Enzymes Selection</h1>
        <section id="methods" class="mb-5">
            <h2>Methods</h2>
            <p>In order to perform a screening of proteins that could perform the same reaction as the selected enzymes for steps 5, 6, 9, 10, and 11, we performed a search using different approaches in different databases. We searched
                for proteins with the same EC number for classification of enzymes as the selected ones in the UniProt and
                Brenda databases. From the InterPro database, we got proteins with the corresponding Pfam Domains. From a
                homology-based approach, we performed a blastp against the NCBI database as well as a blast against the UniProt
                database.</p>
            <p>From these strategies, we created a pool of candidate enzymes for the different steps of the proposed
                biosynthetic pathway. We obtained the peptidic sequence for all the candidates in order to perform the next
                steps for the selection of the next enzymes that will be probed for expected catalytic activity.</p>
            <p>For enzymes 5 and 6, we performed a cluster analysis of the enzymes using the CD-HIT, a program used to reduce
                sequence redundancy and improve the performance of further analyses. We aligned the sequences using the muscle
                algorithm.</p>
            <p>Another clustering strategy was used, based on the WGCNA R package. This is a correlation network analysis
                designed for studying gene coexpression, but the mathematical background was used in this case to analyze and
                cluster the universe of candidate enzymes. We defined the adjacency matrix as the sequence identity value
                between each two enzymes calculated by the seqidentity percent identity function from the bio3d R package.</p>
            <pre><code>ai,j = seqidentity(xi,xj)</code></pre>
            <p>Where a is the adjacency value, seqidentity indicates the function for getting the sequence percentage identity
                between two proteins and xi and xj are the sequences in the i row and j position. This matrix is the size n x n,
                where n is the number of sequences. The distance matrix is defined as follows:</p>
            <pre><code>di,j = 1 - ai,j</code></pre>
            <p>The enzymes were then clustered by the UPGMA (unweighted pair group method with arithmetic mean) algorithm. The
                modules were formed by this methodology by using a constant-height cut strategy.</p>
            <p>The top 20 proteins with the highest identity to the original enzyme were selected for the next steps.
                Additionally, the enzymes that were representatives of a CD-HIT cluster and had an annotation corresponding to
                the expected enzymatic activity were selected for the next steps.</p>
        </section>
        <section id="e5" class="mb-5">
            <h2>E5</h2>
            <p>The next plot shows the distribution of the protein identity to the reference enzyme. We can observe a
                distribution with proteins with an identity value smaller than 0.4, another group of candidate proteins with
                identity values between 0.4 and 0.6, and only a few proteins with greater values.</p>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-1.png"
                    alt="Alternative Enzymes 1" class="img-fluid my-3"></div>
            <p>When visualizing the adjacency matrix with a heatmap, we can observe the formation of clusters of proteins that
                are more similar within the clusters than between clusters.</p>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-2.png"
                    alt="Alternative Enzymes 2" class="img-fluid my-3"></div>
            <p>The clusters formed by the CD-HIT were created with a threshold value of 0.6 and a word size of 4, forming 44
                clusters. In the case of the correlation network analysis, the threshold for the cut was 0.4. The comparison of
                the CD-HIT clusters against the correlation network (StaticAdj) modules is shown in the next figure.</p>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-3.png"
                    alt="Alternative Enzymes 3" class="img-fluid my-3"></div>
            <p>The clusters and modules can be visualized in the gene universe as a network, where the weight of the connections
                corresponds to the adjacency of each pair of proteins and the size of the node corresponds to the identity to
                the reference protein. In the next figures, we can observe the clusters and modules formed by the two methods in
                the protein universe represented as a network, where each color represents a different cluster or module.</p>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-4.png"
                    alt="Alternative Enzymes 4" class="img-fluid my-3"></div>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-5.png"
                    alt="Alternative Enzymes 5" class="img-fluid my-3"></div>
            <p>To show how good the sampling of the universe of candidate proteins was, we show the selected proteins in the
                universe. The blue nodes represent the candidates selected by homology/identity to the reference protein
                sequence, and the orange ones represent the proteins selected because they are CD-HIT cluster representatives.
                In this case, we ended with 56 selected sequences for the next steps. This demonstrates the importance of using
                both approaches, as it provides better sampling of different sites in the universe.</p>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-6.png"
                    alt="Alternative Enzymes 6" class="img-fluid my-3"></div>
        </section>
        <section id="e6" class="mb-5">
            <h2>E6</h2>
            <p>The next plot shows the distribution of the protein identity to the reference enzyme. We can observe a
                distribution with proteins with identity values between 0.55 and 0.70, with only a few proteins having greater
                values.</p>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-7.png"
                    alt="Protein Identity Distribution" class="img-fluid my-3"></div>
            <p>When visualizing the adjacency matrix with a heatmap, we can observe the formation of a big cluster of proteins
                with high identity within the group and only a few sequences outside this big cluster.</p>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-8.png"
                    alt="Adjacency Matrix Heatmap" class="img-fluid my-3"></div>
            <p>The clusters formed by the CD-HIT were created with a threshold value of 0.7 and a word size of 5, resulting in
                39 clusters. In the case of the correlation network analysis, the threshold for the cut was 0.25. The comparison
                of the CD-HIT clusters against the correlation network (StaticAdj) modules is shown in the next figure.</p>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-9.png"
                    alt="CD-HIT vs Correlation Network" class="img-fluid my-3"></div>
            <p>In the next figures, we can observe the clusters and modules formed by the two methods in the protein universe
                represented as a network, where each color represents a different cluster or module.</p>
            <div class="image-placeholder">E6 CD-HIT Clusters</div>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-10.png"
                    alt="CD-HIT Clusters" class="img-fluid my-3"></div>
            <div class="image-placeholder">E6 Correlation Network Modules</div>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-11.png"
                    alt="Correlation Network Modules" class="img-fluid my-3"></div>
            <p>In the next figure, we show the selected proteins in the universe. The blue nodes represent candidates selected
                by homology/identity to the reference protein sequence, and the orange ones represent the proteins selected as
                CD-HIT cluster representatives. In total, 47 enzymes were selected for the next steps.</p>
            <div class="image-placeholder">E6 Selected Enzymes</div>
            <div class="text-center"><img src="../../images/dry lab/alternative enzymes selection/alternative-enzymes-12.png"
                    alt="Selected Enzymes" class="img-fluid my-3"></div>
        </section>
        <section id="e9-e10" class="mb-5">
            <h2>E9 &amp; E10</h2>
            <p>Initially, we proposed that E9 could catalyze the cyclization of 5-aminopentanal to form 1-piperideine. However,
                a paper published this year suggested that this process occurs spontaneously without the need for enzymatic
                activity. As a result, we ceased our search for alternative enzymes for this specific reaction (Lv, et al.,
                2024).</p>
            <p>On the other hand, the case of E10 was more complicated. The same paper (Lv, et al., 2024) remarked that this
                step is one of the key unsolved steps of the pathway, and no one knows which enzyme is involved. Our first
                proposal was a bacterial alternative that catalyzed a similar, but not identical, reaction. Therefore, since we
                were not sure about if that enzyme was even the best choice, our procedure consisted in reading papers to find
                other similar reactions in different organisms. After that, we performed BLAST to include the closer enzymes so
                we could cover many alternatives. In total, we included three distinct groups and their BLAST relatives. We used
                a reductive aminase from Aspergillus oryzae, galactose oxidase, and imine reductase from Fusarium graminearum.
                (Aleku, et al., 2017; Ford, et al., 2020).</p>
        </section>
        <section id="references" class="mb-5">
            <h2>References</h2>
            <ul>
                <li>Aleku, G. A., et al. (2017). A reductive aminase from Aspergillus oryzae. Nature Chemistry, 9(10), 961-969.
                </li>
                <li>Edgar, R. C. (2004). MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic
                    Acids Research, 32(5), 1792-1797.</li>
                <li>Ford, G. J., et al. (2020). Synthesis of protected 3-aminopiperidine and 3-aminoazepane derivatives using
                    enzyme cascades. Chemical Communications, 56(57), 7949-7952.</li>
                <li>Fu, L., et al. (2012). CD-HIT: accelerated for clustering the next-generation sequencing data.
                    Bioinformatics, 28(23), 3150-3152.</li>
                <li>Grant, B. J., et al. (2006). Bio3d: an R package for the comparative analysis of protein structures.
                    Bioinformatics, 22(21), 2695-2696.</li>
                <li>Langfelder, P., &amp; Horvath, S. (2008). WGCNA: an R package for weighted correlation network analysis. BMC
                    Bioinformatics, 9, 1-13.</li>
                <li>Lv, Y., et al. (2024). Metabolome profiling and transcriptome analysis filling the early crucial missing
                    steps of piperine biosynthesis in Piper nigrum L. The Plant Journal, 117(1), 107-120.</li>
            </ul>
        </section>
    </div> <!-- End Body -->

    <footer style="background-color: #F5F5F5;" > <!-- Footer -->
        <div class="container">
            <h2 class="text-center" style="padding-top: 50px;">Partners</h2>
            <p>The companies and organizations that supported us:</p>
        </div>
        
        <div class="footer-carousel">
            <div class="carousel-wrapper">
                <div class="carousel-track">
                    <img src="../../images/sponsors/climaton.png" alt="climaton">
                    <img src="../../images/sponsors/Emblem_of_the_All-China_Youth_Federation.svg.png" alt=" Youth Federation">
                    <img src="../../images/sponsors/Escudo-UNAM-escalable.png" alt="Escudo de la UNAM">
                    <img src="../../images/sponsors/igemstartups.jpeg" alt="igem startups">
                    <img src="../../images/sponsors/uniceflogo.png" alt="unicef logo">
                    <img src="../../images/sponsors/ciencias.webp" alt="Logotipo de la Facultad de Ciencias de la UNAM">
                    <!-- Repetimos las mismas imÃ¡genes para efecto de bucle -->
                    <img src="../../images/sponsors/climaton.png" alt="climaton">
                    <img src="../../images/sponsors/Emblem_of_the_All-China_Youth_Federation.svg.png" alt=" Youth Federation">
                    <img src="../../images/sponsors/Escudo-UNAM-escalable.png" alt="Escudo de la UNAM">
                    <img src="../../images/sponsors/igemstartups.jpeg" alt="igem startups">
                    <img src="../../images/sponsors/uniceflogo.png" alt="unicef logo">
                    <img src="../../images/sponsors/ciencias.webp" alt="Logotipo de la Facultad de Ciencias de la UNAM">
                </div>
            </div>
        </div>
    </footer>

    <script src="../../js/bootstrap.js"></script>
</body>
</html>