<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Design - Alebricks</title>
    <link href="../../css/bootstrap.css" rel="stylesheet">
    <link href="../../css/general.css" rel="stylesheet">
</head>
<body>
    <div style="background-color: #F8F9FA;">
        <div class="container"> <!-- NavBar-->
            <nav class="navbar navbar-expand-lg fixed-top bg-body-tertiary">
                <div class="container-fluid">
                    <a class="navbar-brand" href="../../index.html">Home</a>
                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
                        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                        <span class="navbar-toggler-icon"></span>
                    </button>
                    <div class="collapse navbar-collapse" id="navbarSupportedContent">
                        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
                            <li class="nav-item">
                                <!-- <a class="nav-link active" aria-current="page" href="#">Home</a> -->
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">About</a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../about/members.html"> Members</a></li>
                                    <li><a class="dropdown-item" href="../about/mission_vision.html">Mission & Vision</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Project
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../project/description.html">Description</a></li>
                                    <li><a class="dropdown-item" href="../project/roadmap.html">Roadmap</a></li>
                                    <li><a class="dropdown-item" href="../project/engineering_success.html">Engineering success</a></li>
                                    <li><a class="dropdown-item" href="../project/business_model.html">Business model</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Safety
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../safety/biosafety.html">Biosafety & Biosecurity</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Wet & Field Lab
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="./contribution.html">Contribution</a></li>
                                    <li><a class="dropdown-item" href="./design.html">Design</a></li>
                                    <li><a class="dropdown-item" href="./citotoxicity.html">Citotoxicity</a></li>
                                    <li><a class="dropdown-item" href="./phytotoxicity.html">Phytotoxicity</a></li>
                                    <li><a class="dropdown-item" href="./field test.html">Field Tests</a></li>
                                    <li><a class="dropdown-item" href="./notebook.html">Notebook</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Dry Lab
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../dry lab/Overview.html">Overview</a></li>
                                    <li><a class="dropdown-item" href="../dry lab/genome_annotation.html">Genome annotation</a></li>
                                    <li><a class="dropdown-item" href="../dry lab/differential-expression-analysis.html">Differential expression analysis</a></li>
                                    <li><a class="dropdown-item" href="../dry lab/docking.html">Docking</a></li>
                                    <li><a class="dropdown-item" href="../dry lab/alternative-enzymes-selection.html">Alternative enzymes selection</a></li>
                                    <li><a class="dropdown-item" href="../dry lab/docking-mds.html">Docking and MDS</a></li>
                                    <li><a class="dropdown-item" href="../dry lab/model.html">Math-Model</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Human Practices
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../human practices/human practices.html">Human Practices</a></li>
                                    <li><a class="dropdown-item" href="../human practices/human.html">Human</a></li>
                                    <li><a class="dropdown-item" href="../human practices/Diversity and Inclusivity.html">Diversity and Inclusivity</a></li>
                                    <li><a class="dropdown-item" href="../human practices/Education.html">Education</a></li>
                                    <li><a class="dropdown-item" href="../human practices/Philosophy and episthemology.html">Philosophy and episthemology</a></li>
                                    <li><a class="dropdown-item" href="../human practices/Reach.html">Reach</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
                                    Arts & Design
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../arts and design/gallery.html">Gallery</a></li>
                                    <li><a class="dropdown-item" href="../arts and design/videogame.html">Videogame</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
                                    Aknowledgements
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="../aknowledgements/attributions.html">Attributions</a></li>
                                    <li><a class="dropdown-item" href="../aknowledgements/crowdfunding.html">Crowdfunding & Sponsors</a></li>
                                </ul>
                            </li>
                            <li class="nav-item dropdown">
                                <a class="nav-link dropdown-toggle disabled" href="#" role="button" data-bs-toggle="dropdown"
                                    aria-expanded="false">
                                    Sustainable development
                                </a>
                                <ul class="dropdown-menu">
                                    <li><a class="dropdown-item" href="#">Ecological Niche Modeling</a></li>
                                    <li><a class="dropdown-item" href="#">SDG's</a></li>
                                    <li><a class="dropdown-item" href="#">Negative Impact</a></li>
                                    <li><a class="dropdown-item" href="#">Why to protect maize?</a></li>
                                    <li><a class="dropdown-item" href="#">Climaton</a></li>
                                </ul>
                            </li>
                        </ul>
                    </div>
                </div>
            </nav>
        </div>
    </div> <!-- End Container NavBar-->

    <div class="container my-5 pt-4"> <!-- Body -->
        <div class="card my-4">
            <div class="card-header" id="introduction">
                <h2>Design Introduction</h2>
            </div>
            <div class="card-body">
                <p class="card-text">This project is structured into two major phases. As a consequence, we use different
                    construct designs for each. The first phase focuses on research aimed at identifying the enzymes that are
                    likely to catalyze the expected reactions, based on a highly refined computational analysis that can be
                    consulted in the DryLab section. This stage involves proposing candidate enzymes and conducting individual
                    laboratory tests for each one. To support this effort, we partnered with Ginkgo Bioworks, a company that
                    sponsored the DNA synthesis of individual plasmids, each encoding one of the enzymes in the pathway.
                    Additionally, they provided experimental testing of these enzymes to validate their activity.</p>
                <p class="card-text">The second phase focuses on the reconstruction of the entire metabolic pathway in yeast.
                    Based on our mathematical model, the pathway is expected to be more efficient if all the necessary enzymes
                    are integrated into a single <em>Saccharomyces cerevisiae</em> strain, rather than distributed across
                    multiple strains with metabolite transfers between them. This consolidated approach allows for better
                    control and optimization of the pathway, reducing inefficiencies. Therefore, our next steps will focus on
                    refining this design for optimal performance.</p>
                <p class="card-text"><em>Saccharomyces cerevisiae</em> is an ideal chassis for this project due to several key
                    factors. It can achieve high cell density in cultures and is a well-characterized eukaryotic organism, which
                    makes it highly amenable to genetic manipulation. Its ability to perform post-translational modifications
                    ensures proper folding and functioning of complex eukaryotic proteins, which is essential for the enzymes
                    involved in our pathway. Additionally, <em>S. cerevisiae</em>'s robustness under various environmental
                    conditions, its ease of large-scale cultivation, and its capacity to produce high yields of recombinant
                    products make it a cornerstone of industrial biotechnology. The extensive genetic tools and resources
                    available for <em>S. cerevisiae</em> further enhance its flexibility and efficiency in synthetic biology
                    applications, making it the perfect platform for our pathway integration and optimization.</p>
            </div>
        </div>

        <div class="card my-4">
            <div class="card-header" id="phase-one"><h2>Phase I: Designs for Testing</h2></div><div class="card-body"><p class="card-text">In the initial phase of this project, we proposed a set of 11 enzymes that required individual testing to verify their potential to catalyze the desired reactions. To facilitate this, Ginkgo Bioworks offered their support by synthesizing constructs capable of expressing these enzymes in different yeast strains. Since we do not have permission from Ginkgo to share specific details about their constructs, this section does not include them. However, the constructs were transcriptional units consisting of a constitutive promoter, followed by our coding sequences (CDS), and a terminator.</p><p class="card-text">Out of the proposed enzymes, we were particularly interested in testing only those that had not been previously validated in the laboratory, in order to explore their functions. As a result, the CDSs utilized corresponded to enzymes 5, 6, 7, 8, 9, and 10. Each of these CDSs was optimized for yeast codon usage to ensure efficient expression. Our team held several meetings with Ginkgo to assist them in sourcing the required metabolites for each enzymatic reaction.</p><p class="card-text">However, Ginkgo’s experimental assays did not yield conclusive results. Thus, it became our responsibility to perform the enzyme testing ourselves. To proceed, we carried out a secondary enzyme selection process, which can be reviewed in the "Alternative Enzymes Selection" section. This refined selection was further validated through docking analysis and molecular dynamics simulations, as detailed in their respective sections.</p><p class="card-text">Following this refinement, Ginkgo offered to synthesize constructs for each of the alternative enzymes we proposed, resulting in a total of 127 plasmids. These constructs were designed for genomic integration, with codon optimization tailored for yeast. Ginkgo sent these reagents to our university for further testing. It is important to emphasize that the construct designs were entirely the work of Ginkgo, and our contribution was limited to providing the CDS sequences.</p></div>
        </div>
        <div class="card my-4">
            <div class="card-header" id="phase-two">
                <h2 style="background-color: yellow;">Phase II: Designs for Production</h2>
            </div>
            <div class="card-body">
                <p class="card-text">Metabolite synthesis by enzymatic reactions is a complex process for all living organisms on Earth. In nature, the synthesis machinery is a system composed of numerous components and variables that influence the final outcome. One of the most important factors to consider in such mechanisms is the time required to complete each reaction.</p>
                <p class="card-text">For example, during a metabolic pathway, if an enzyme is highly expressed but its corresponding substrate is not present in the necessary quantity, the enzyme will not be able to function. As a result, the organism will suffer from inefficient use of amino acids that could have been allocated to more essential tasks. Due to this, evolution has favored the development of various tools that allow cells to optimize when and what proteins are translated to maintain homeostasis. One such tool is reflected in the configuration of transcription networks. A common arrangement in metabolic transcription networks to address this challenge is the creation of temporal programs. These programs allow proteins to be translated approximately at the same time that their corresponding metabolites accumulate. (Alon, 2019).</p>
                <p class="card-text">This is often achieved by the sequential activation of the promoters involved in the pathway, where each promoter responds to the same transcription factor but with varying affinities. This means that some promoters are activated at lower concentrations of the transcription factor, leading to a staggered expression of genes as the activator concentration increases. These configurations are known as Single Input Modules (SIM), where a set of genes is regulated by a single transcription factor. (Alon, 2019).</p>
                <p class="card-text">In synthetic biology, expression systems are often tasked with producing proteins that offer no immediate benefit to the host organism's survival. When multiple genes need to be expressed simultaneously, as in the case of metabolic pathways, the resource-saving mechanisms typically found in natural systems are usually bypassed. Given that our project involves such a pathway, we hypothesize that implementing an artificial temporal program could optimize the production of the necessary proteins. More importantly, simultaneous expression of multiple transgenic proteins could exacerbate pleiotropic effects and increase the risk of autotoxicity from accumulated metabolites. Additionally, issues such as protein aggregation are commonly observed in these scenarios, further underscoring the need for controlled, staggered expression.</p>
            </div>
        </div>
        <div class="card my-4">
            <div class="card-header" id="phase-two">
                <h2 style="background-color: yellow;">Phase II: Designs for Production</h2>
            </div>
            <div class="card-body">
                <p class="card-text"><strong>A design that mimics nature to optimize productivity</strong></p>
                <p class="card-text">We discovered that three of the enzymes involved in piperamide synthesis are overexpressed in
                    <em>Piper nigrum</em>mature fruit. These three proteins participate in the final stages of the pathway, with one
                    of them being Piperamide Synthase, which performs the last step in piperamide synthesis. The fact that evolution
                    has seemingly developed a mechanism to separate the expression of these enzymes over time provided us with a
                    valuable insight into the importance of timing control in this molecular process. Additionally, we were familiar
                    with a known mechanism to achieve this—Single Input Modules (SIMs)—which gave us a solid foundation for our
                    design.</p>
                <p class="card-text">However, implementing a system with 11 promoters activated by the same transcription factor,
                    but with different affinities, presents a significant challenge. Moreover, using 11 transcriptional units would
                    drastically increase the number of DNA parts required, adding complexity to the system and increasing the
                    likelihood of failure.</p>
                <p class="card-text">To address this, we devised a completely artificial temporal program using two promoters: a
                    constitutive promoter to manage the expression of the first block of proteins, and a galactose inducible
                    promoter (pGAL) to activate the production of the final proteins—those that we identified as being overexpressed
                    in the mature fruit—once their substrate demand could be met.</p>
                <p class="card-text">This approach only partially solved the problem, as the number of required promoters,
                    terminators, and ribosome binding sites would still be substantial. While using bacteria as the expression
                    system could have addressed this through polycistronic constructs, bacteria are not entirely reliable for
                    expressing eukaryotic proteins. Given that we were working with 11 proteins, the probability of encountering
                    expression difficulties was high. Therefore, we opted for yeast as our expression system.</p>
                <p class="card-text">At this point, we leveraged a powerful tool: 2A peptides. These peptide sequences can be
                    inserted between two genes, linking them and enabling co-expression through a single promoter. Essentially, they
                    function like polycistronic units. We decided to include them in our design. (Liu, et al., 2017).</p>
                <p class="card-text">The PTE2A construct has been shown to be highly efficient for quad-cistronic constructs in
                    mice, as it successfully expressed a good amount of proteins. However, it did exhibit some challenges in
                    translating the last fragments of the transcriptional unit. (Liu, et al., 2017). Interestingly, we propose that
                    this imperfection could be useful for fine- tuning the timing of the temporal program.</p>
                <p class="card-text">We established the order of protein expression in a manner that reflects the synthesis steps of
                    piperamide, where each enzyme's position is determined by the step it performs in the pathway. The pathway
                    involves two sub- pathways, each starting with an amino acid (phenylalanine and lysine) and converging at
                    Piperamide Synthase.</p>
                <p class="card-text">By combining a constitutive and an inducible promoter, along with adjustments based on the
                    positioning of genes in polycistronic constructs, we have developed a robust mechanism to control the timing of
                    protein production. We organized the genes into three constructs, two of which are managed by a constitutive
                    promoter and one by an inducible promoter. The gene positions were chosen carefully, considering the decrease in
                    expression levels as the distance from the promoter increases. That is also why they were placed into three
                    different transcriptional units, each containing four genes, as including more genes per unit could lead to
                    suboptimal expression.</p>
                <p class="card-text">In the following image we represent above the pathway and below the final design of the
                    artificial polycistronic constructs. It involves two sub-pathways: the phenylalanine sub-pathway, represented by
                    green genes, and the lysine sub-pathway, represented by red genes. These two pathways converge when Piperamide
                    Synthase catalyzes the final reaction to form piperamide. Each gene in the construct is separated by a 2A
                    peptide, which allows the co-expression of multiple genes during translation. Although not depicted in the
                    figure, these peptides ensure that the genes are expressed as individual proteins. The decision to use both a
                    constitutive and an inducible promoter was made to allow for precise control over the timing of gene expression.
                    The inducible promoter can be activated at any chosen time, enabling the creation of a temporal expression
                    program for optimal production.</p>
            </div>
        </div>
        <div class="card my-4">
            <p class="card-text">We revised the design by reducing the constructs from three to two, each containing the coding sequences (CDS) for four genes. The genes are separated by 2A peptides, which allow co-expression of multiple proteins during translation.
            </p>
            <img src="../../images/wet and field lab/design/design-1.png" class="img-fluid mb-3" alt="Genetic constructs diagram">
            <p class="figure-text"><strong>Figure 2:</strong> The proposed temporal program is shown at the top. Each number in a yellow circle represents the order the gene is going to be expressed. To accomplish that, we combined the use of a constitutive promoter for the proteins in the blue block and a galactose induced promoter in the yellow block. This last block contains the three proteins that we found out are overexpressed in mature black pepper fruit, except for Lys4, which was not found in the same amount. To polish even more the timing inside each block, we took advantage of the production differences that the 2A peptides imply in artificial poli- cistronic constructions (read the Model section). The bottom shows the design of the artificial polycistronic constructs. Green genes carry out the phenylalanine subpathway, whereas red ones are implied in the lysine subpathway. Both of them come together when piperamide synthase catalyzes the final reaction to form piperamide. Between each gene exists a 2A Peptide that separates two genes during translation (not in the figure). The decision of using a constitutive and an inducible promoter         was made because the inducible one can be expressed at any desired time, allowing to create a temporal program.</p>
        </div>
        <div class="card my-4">
            <div class="card-body">
                <p class="card-text"><strong>Problems with our design</strong></p>
                <p class="card-text">The first issue we encountered was discovering that <em>Saccharomyces cerevisiae</em> could
                    naturally produce p-coumaric acid using its own endogenous enzymes (Rodríguez et al., 2015). Since p-coumaric
                    acid is a key intermediate in our pathway, we chose to streamline the design by removing the first two enzymes
                    (1 and 2) to avoid redundancy. Furthermore, a recently published paper revealed that reaction 9 in our pathway
                    was, in fact, a spontaneous process and did not require any enzymatic action (Lv, et al., 2024). As a result, we
                    also removed the enzyme originally assigned to this step. This left us with a total of 8 enzymes, a significant
                    reduction from the 11 initially proposed.</p>
                <p class="card-text">Additionally, the original construct was designed using classical cloning methods with Type II
                    restriction enzymes—a technique that has become quite outdated by modern synthetic biology standards. To
                    streamline the process and align with current practices, we decided to transition to a methodology based on
                    Golden Gate Assembly.</p>
                <p class="card-text">To further enhance the stability of our constructs, we opted for genome-integrating plasmids
                    instead of autonomous plasmids. This approach not only improves the stability of gene expression but is also
                    crucial for biosafety, as it reduces the risk of horizontal gene transfer from these engineered strains compared
                    to using free plasmids.</p>
                <p class="card-text"><strong>Revised and optimized designs</strong></p>
                <p class="card-text">Following the investigation and optimization process of our metabolic pathway, we have proposed
                    a new design that consolidates the genetic constructs from three to two. Each construct will contain the coding
                    sequences (CDS) for four genes, with their expression controlled by a modified pGAL1 promoter and flanked by
                    three 2A peptides between the CDS of the proteins, as illustrated in the figure below.</p><img
                    src="../../images/wet and field lab/design/desing-2.png" class="img-fluid mb-3"
                    alt="Genetic constructs diagram">
                <p class="figure-text"><strong>Figure 2:</strong> The diagram depicts the composition of the two proposed genetic
                    constructs. Construct 1 contains the CDS for the enzymes of the acyl donor pathway, while Construct 2 includes
                    those of the amine donor pathway. Both constructs share the same promoter, 2A peptides, and terminator, but the
                    genomic insertion site (not shown) will differ.</p>
                <p class="card-text">The <strong>GAL1 promoter</strong> (Figure 3) was modified based on the BBa_J63006 part, which
                    is similar to the wild-type promoter, by adding 5' and 3' ends compatible with Golden Gate assembly, as used in
                    the Open Yeast Collection. Additionally, the binding sites for the MIG1 protein, which acts as a repressor in
                    the presence of glucose, were deleted (Figure 4). However, the four binding sites for GAL4, the inducer in the
                    presence of galactose, were retained.</p>
                <p class="card-text">We hypothesize that this will allow us to activate the expression of the two constructs by
                    adding galactose to the medium during the stationary phase of growth, while continuing to use glucose as the
                    primary carbon source. This approach will ensure that the concentrations of galactose are not impacted, nor will
                    the yeast's metabolic capacity be compromised by the synthesis of proteins for galactose metabolism.</p>
                <p class="card-text">The endogenous GAL1 promoter will retain the MIG1 binding regions, and thus remain inhibited in
                    the presence of glucose, which is beneficial for controlling the expression of our genetic constructs with a
                    relatively low-cost inducer like galactose.</p><img
                    src="../../images/wet and field lab/design/desing-3.png" class="img-fluid mb-3"
                    alt="Modified GAL1 promoter diagram">
                <p class="figure-text"><strong>Figure 3:</strong> Activation and repression in the UASG region of the wild-type GAL1
                    yeast promoter under different conditions: a) Glucose present, galactose absent (repressed); b) Glucose and
                    galactose both present (repressed); c) Glucose absent, galactose present (active).</p>
                <p class="card-text">In addition to the deletions of the MIG1 binding sites, the promoter sequences were reviewed to
                    eliminate any restriction sites incompatible with the RFC10 standard. Complementary ends were also added to the
                    promoters to ensure compatibility with other parts of the Open Yeast Collection. We synthesized the promoter
                    sequences accordingly (registered as part BBa_K5479011). To evaluate its performance, we synthesized the GFP
                    enzyme CDS with complementary ends for Golden Gate assembly with other parts from the Open Yeast Collection (Fig
                    5).</p><img src="../../images/wet and field lab/design/desing-4.png" class="img-fluid mb-3"
                    alt="MIG1 binding sites removed">
                <p class="figure-text"><strong>Figure 4:</strong> Comparison of the expected behavior of the wild-type and modified
                    (-MIG1) promoter in the presence of glucose and galactose.</p><img
                    src="../../images/wet and field lab/design/desing-5.png" class="img-fluid mb-3"
                    alt="Golden Gate Assembly">
                <p class="figure-text"><strong>Figure 5:</strong> Schematic of the proposed assembly of the modified pGAL1 promoter
                    together with GFP, using Golden Gate.</p>
                <p class="card-text">Alongside the promoter, we also designed and synthesized a part containing the 2A peptide
                    sequences. These sequences were optimized with complementary ends for Golden Gate assembly, ensuring
                    compatibility with other parts used in the Open Yeast Collection. The purpose of this element is to facilitate
                    the use of 2A peptides in assembling our final construct (Fig 6). The selected CDS from the pool synthesized by
                    Ginkgo Bioworks will be amplified using primers that add these complementary ends.</p><img
                    src="../../images/wet and field lab/design/desing-6.png" class="img-fluid mb-3"
                    alt="2A Peptides for Golden Gate">
                <p class="figure-text"><strong>Figure 6:</strong> Schematic representation of the designed and synthesized part
                    containing the PSA, T2A, and E2A peptides with complementary ends for Golden Gate assembly.</p>
            </div>
        </div>
    </div> <!-- End Body -->

    <footer style="background-color: #F5F5F5;" > <!-- Footer -->
        <div class="container">
            <h2 class="text-center" style="padding-top: 50px;">Partners</h2>
            <p>The companies and organizations that supported us:</p>
        </div>
        
        <div class="footer-carousel">
            <div class="carousel-wrapper">
                <div class="carousel-track">
                    <img src="../../images/sponsors/climaton.png" alt="climaton">
                    <img src="../../images/sponsors/Emblem_of_the_All-China_Youth_Federation.svg.png" alt=" Youth Federation">
                    <img src="../../images/sponsors/Escudo-UNAM-escalable.png" alt="Escudo de la UNAM">
                    <img src="../../images/sponsors/igemstartups.jpeg" alt="igem startups">
                    <img src="../../images/sponsors/uniceflogo.png" alt="unicef logo">
                    <img src="../../images/sponsors/ciencias.webp" alt="Logotipo de la Facultad de Ciencias de la UNAM">
                    <!-- Repetimos las mismas imágenes para efecto de bucle -->
                    <img src="../../images/sponsors/climaton.png" alt="climaton">
                    <img src="../../images/sponsors/Emblem_of_the_All-China_Youth_Federation.svg.png" alt=" Youth Federation">
                    <img src="../../images/sponsors/Escudo-UNAM-escalable.png" alt="Escudo de la UNAM">
                    <img src="../../images/sponsors/igemstartups.jpeg" alt="igem startups">
                    <img src="../../images/sponsors/uniceflogo.png" alt="unicef logo">
                    <img src="../../images/sponsors/ciencias.webp" alt="Logotipo de la Facultad de Ciencias de la UNAM">
                </div>
            </div>
        </div>
    </footer>

    <script src="../../js/bootstrap.js"></script>
</body>
</html>